docs/src/index.md

```diff
@@ -1,124 +1,53 @@
 # NeuralNetDiffEq
 
-The repository is for the development of neural network solvers of differential equations such as physics-informed
-neural networks (PINNs) and deep BSDE solvers. It utilizes techniques like deep neural networks and neural 
-stochastic differential equations to make it practical to solve high dimensional PDEs efficiently through the
-likes of scientific machine learning (SciML).
+NeuralNetDiffEq.jl consists neural network solvers for differential equations, such as physics-informed neural networks (PINNs), and deep BSDE solvers.
+NeuralNetDiffEq utilizes techniques like deep neural networks and neural stochastic differential equations to solve high dimensional PDEs efficiently as a part of scientific machine learning (SciML).
 
-# Examples
+## Solving ODEs with Neural Networks
 
-## DeepBSDE Solver
+For ODEs, [see the DifferentialEquations.jl documentation](http://docs.juliadiffeq.org/dev/solvers/ode_solve#NeuralNetDiffEq.jl-1)
+for the `nnode(chain,opt=ADAM(0.1))` algorithm, which takes in a Flux.jl chain
+and optimizer to solve an ODE. This method is not particularly efficient, but
+is parallel. It is based on the work of:
 
-## Solving a 100 dimensional Hamilton-Jacobi-Bellman Equation
+[Lagaris, Isaac E., Aristidis Likas, and Dimitrios I. Fotiadis. "Artificial neural networks for solving ordinary and partial differential equations." IEEE Transactions on Neural Networks 9, no. 5 (1998): 987-1000.](https://arxiv.org/pdf/physics/9705023.pdf)
 
-In this example we will solve a Hamilton-Jacobi-Bellman equation of 100 dimensions.
-The Hamilton-Jacobi-Bellman equation is the solution to a stochastic optimal
-control problem. Here, we choose to solve the classical Linear Quadratic Gaussian
-(LQG) control problem of 100 dimensions, which is governed by the SDE
-`dX_t = 2sqrt(λ)c_t dt + sqrt(2)dW_t` where `c_t` is a control process. The solution
-to the optimal control is given by a PDE of the form:
+## Solving Kolmogorov Equations with Neural Networks
 
-![HJB](https://user-images.githubusercontent.com/1814174/63213366-b1817b80-c0d9-11e9-99b2-c8c08b86d2d5.PNG)
+A Kolmogorov PDE is of the form :
 
-with terminating condition `g(X) = log(0.5f0 + 0.5f0*sum(X.^2))`. To solve it
-using the `TerminalPDEProblem`, we write:
+![KPDE](https://raw.githubusercontent.com/ashutosh-b-b/Kolmogorv-Equations-Notebook/master/KolmogorovPDEImages/KolmogorovPDE.png)
 
-```julia
-d = 100 # number of dimensions
-X0 = fill(0.0f0,d) # initial value of stochastic control process
-tspan = (0.0f0, 1.0f0)
-λ = 1.0f0
-
-g(X) = log(0.5f0 + 0.5f0*sum(X.^2))
-f(X,u,σᵀ∇u,p,t) = -λ*sum(σᵀ∇u.^2)
-μ_f(X,p,t) = zero(X)  #Vector d x 1 λ
-σ_f(X,p,t) = Diagonal(sqrt(2.0f0)*ones(Float32,d)) #Matrix d x d
-prob = TerminalPDEProblem(g, f, μ_f, σ_f, X0, tspan)
-```
+Considering S be a solution process to the SDE:
```

Comment:
Consider S as the solution process of the SDE:

etc. there's more like this, so these edits probably need a second read

Reply:
I just moved old readme around. I will just rewrite them then.